frontend/src/layout/navigation/navigationLogic.ts

```diff
@@ -3,18 +3,18 @@ import api from 'lib/api'
 import { systemStatusLogic } from 'scenes/instance/SystemStatus/systemStatusLogic'
 import { userLogic } from 'scenes/userLogic'
 import { navigationLogicType } from './navigationLogicType'
-import { UserType } from '~/types'
+import { SystemStatus, UserType } from '~/types'
 
-export const navigationLogic = kea<navigationLogicType<UserType>>({
+export const navigationLogic = kea<navigationLogicType<UserType, SystemStatus>>({
     actions: {
-        setMenuCollapsed: (collapsed) => ({ collapsed }),
+        setMenuCollapsed: (collapsed: boolean) => ({ collapsed }),
         collapseMenu: () => {},
```

Comment:
NIT: make this `true`?

Reply:
The `collapseMenu` ? Seemed a bit out of scope...